content/orangeciplugins/tencentyun-coscmd/index.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2019-08-29T00:00:00+00:00
+title: tencentyun-coscmd
```

Comment:
This should be a human readable title, not some kind of slug.

Reply:
fixed